File: Examples/CSharp/AsposePDF/Annotations/DeleteParticularAnnotation.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.Annotations
{
    public class DeleteParticularAnnotation
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Annotations();

            //open document
            Document pdfDocument = new Document(dataDir + "DeleteParticularAnnotation.pdf");

            //delete particular annotation
            pdfDocument.Pages[1].Annotations.Delete(1);

            //save updated document
            pdfDocument.Save(dataDir + "DeleteParticularAnnotation_out.pdf");

        }
    }
}

File: Examples/VisualBasic/AsposePDF/Annotations/DeleteParticularAnnotation.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf
Namespace AsposePDF.Annotations
    Public Class DeleteParticularAnnotation
        Public Shared Sub Run()
            ' ExStart:DeleteParticularAnnotation
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Annotations()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("DeleteParticularAnnotation.pdf"))

            ' Delete particular annotation
            pdfDocument.Pages(1).Annotations.Delete(1)

            dataDir = dataDir & Convert.ToString("DeleteParticularAnnotation_out_.pdf")
            ' Save updated document
            pdfDocument.Save(dataDir)
            ' ExEnd:DeleteParticularAnnotation
            Console.WriteLine(Convert.ToString(vbLf & "Particular annotation deleted successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace
